.github/workflows/markdowner.py

```diff
@@ -184,6 +184,8 @@ def perform_edits(content, flags, filename):
             print(f"{filename}: Added newline before EOF")
 
     if flags.codeblocks or flags.all:
+        replacements = {"```\n```command": "```\n\n```command"}
```

Comment:
won't it add newline symbol each run?

Reply:
@mineralsfree no, after inserting the second newline, it no longer matches.

Reply:
When editing this script, if you're not careful about which string you put as the replacement it can happen, but then the script won't work, it will hang forever because it will keep replacing and replacing and replacing.